Automattic/jetpack
# Blaze: remove old unused files

## Proposed changes:

We now handle the editor via the Blaze package.

### Other information:


## Jetpack product discussion
* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Go to Posts > Add New
* Publish a new post
* Check the look of the Blaze post-publish panel.
